index.html

```diff
@@ -1831,322 +1831,291 @@ <h2>Proof</h2>
+    <section>
         <h2>
-  Contexts
+JSON
         </h2>
 
         <p>
-  When two software systems need to exchange data, they need to use terminology
-  and a protocol that both systems understand. The <code>@context</code>
-  property ensures that two systems operating on the same DID document are using
-  mutually agreed terminology.
+When producing and consuming DID documents that are in plain JSON (as noted by
+the document metadata), the following rules MUST be followed. 
         </p>
 
-        <p>
-  <a>DID documents</a> MUST include the <code>@context</code> property.
-        </p>
+        <section>
+            <h3>Production</h3>
 
-        <p class="note" title="The JSON-LD Context">
-  More information about the
-  <a href="https://www.w3.org/TR/json-ld/#the-context">JSON-LD Context</a> in
-  general can be found in the [[JSON-LD]] specification.
-        </p>
+            <p>
+A <a>DID document</a> MUST be a single <a data-cite="RFC8259#section-4">JSON object</a> conforming to [[!RFC8259]]. All
+top-level properties of the <a>DID document</a> MUST be represented by using the
+property name as the name of the member of the JSON object. The values of
+properties of the data model described in Section <a href="#data-model"></a>, 
+including all extensions, MUST be encoded in JSON[[RFC8259]] by mapping property
+values to JSON types as follows:
+            </p>
+
+            <p class="issue" data-number="204">
+In this section we use the term "property name" to refer to the string that represents
+the property itself, but this specification still needs to define a concerete term for
+such aspects of a property of a DID document.
+            </p>
+
+            <ul>
+                <li>
+Numeric values representable as IEEE754 MUST be represented as a <a data-cite="RFC8259#section-6">Number type</a>.
+                </li>
+                <li>
+Boolean values MUST be represented as a <a data-cite="RFC8259#section-3">Boolean literal</a>.
+                </li>
+                <li>
+Sequence value MUST be represented as an <a data-cite="RFC8259#section-5">Array type</a>.
+                </li>
+                <li>
+Unordered sets of values MUST be represented as an <a data-cite="RFC8259#section-5">Array type</a>.
+                </li>
+                <li>
+Sets of properties MUST be represented as an <a data-cite="RFC8259#section-4">Object type</a>.
+                </li>
+                <li>
+Empty values MUST be represented as a <a data-cite="RFC8259#section-3">null literal</a>.
+                </li>
+                <li>
+Other values MUST be represented as a <a data-cite="RFC8259#section-7">String type</a>.
+                </li>
+            </ul>
+
+            <p class="issue" data-number="204">
+An "empty" value is not specified by this document. It seems to imply a null
+value, but this is unclear.
+            </p>
+
+            <p>
+All applicable properties of the <a>DID document</a> MUST be included in
```

Comment:
```suggestion
All mandatory properties of the <a>DID document</a> MUST be included in
```

Not sure what "applicable" is attempting to imply? Feels like a no-op -- "The things that apply to the DID Document must be included in the DID Document" -- is that's what's being said?

Reply:
I'll remove the qualifier. If the group wants to define a qualifier due to discussion on issue #204 then it can be added back in.